bundle/pom.xml

```diff
@@ -7,100 +7,111 @@
+                        <id>bnd-process</id>
                         <goals>
-                            <goal>manifest</goal>
+                            <goal>bnd-process</goal>
                         </goals>
-                        <phase>process-classes</phase>
                         <configuration>
-                            <exportScr>true</exportScr>
-                            <instructions>
-                                <_dsannotations>*</_dsannotations>
-                                <_metatypeannotations>*</_metatypeannotations>
-                            </instructions>
+                            <bnd><![CDATA[
+Bundle-Name: ACS AEM Commons - Core
+Bundle-Activator: com.adobe.acs.commons.util.impl.Activator
+Bundle-SymbolicName: com.adobe.acs.acs-aem-commons-bundle
+Bundle-Category: asset-share-commons
+Import-Package: com.adobe.cq.dialogconversion;resolution:=optional,
+                com.amazonaws.*;resolution:=optional, <!-- only required for MCP S3 Ingestor --> \
+                com.github.jknack.handlebars;resolution:=optional, <!-- only required for Report Builder --> \
+                com.day.cq.replication;version="[6.4,7)", <!-- using a wider version range for forward compatibility --> \
+                org.apache.poi.ss.usermodel;version="[3.0,6)",  <!-- using a wider version range for forward compatibility --> \
+                org.apache.poi.ss.util;version="[3.0,6)", \
+                org.apache.poi.xssf.usermodel;version="[2.0,6)", \
+                twitter4j*;version="[3.0.5,4)";resolution:=optional, \
+                org.apache.sling.xss;version="[1.1,3)", <!-- using a wider version range for forward compatibility --> \
+                com.github.benmanes.caffeine*;resolution:=optional, \
+                com.fasterxml.jackson.core;version="[2.8,3)";resolution:=optional, \
+                com.fasterxml.jackson.databind;version="[2.8,3)";resolution:=optional, \
+                <!-- wider version range: use 2.1 as AEM 6.4 still uses it, 2.2 is for AEM 6.5 --> \
+                <!-- required for jwt library used by Adobe I/O APIs --> \
+                javax.xml.bind;version="[2.1,3)", \
+                <!-- this explicit import can be removed when we no longer support \
+                a version of AEM that supports java versions before 11 --> \
+                javax.annotation;version=0.0.0, \
+                !com.google.errorprone.annotations, \
+                !com.google.errorprone.annotations.concurrent, \
+                !org.bouncycastle.jce.*, \
+                !org.checkerframework.checker.nullness.qual, \
+                !android.util.*, \
+                <!-- shaded libraries --> \
+                !com.google.common.*, \
+                !com.google.thirdparty.*, \
+                !com.jcraft.*, \
+                !io.jsonwebtoken.*, \
+                *
+-exportcontents: ${packages;VERSIONED}
+-snapshot: ${tstamp;yyyyMMddHHmmssSSS}
```

Comment:
Probably we can stay with SNAPSHOT as recommended in https://github.com/bndtools/bnd/tree/master/maven/bnd-maven-plugin#reproducible-builds.